File: Python/fizzbuzz2.py
LIMIT = 100

d = {
    (False, False): 'FizzBuzz',
    (False, True): 'Fizz',
    (True, False): 'Buzz',
    (True, True): None,
}

for i in range(1, LIMIT + 1):
    print(d[bool(i % 3), bool(i % 5)] or i, end='')
    if i < LIMIT:
        print(', ', end='')

print()

File: Ruby/fizzbuzz2.rb
LIMIT = 100

a = ['FizzBuzz', '', '', 'Fizz', '', 'Buzz', 'Fizz',
     '', '', 'Fizz', 'Buzz', '', 'Fizz', '', '']

(1..LIMIT).each do |i|
  s = a[i % 15]
  print s.length.zero? ? i : s
  print ", " if i < LIMIT
end

puts
